Documentation/_index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: etcd version ___
+weight: 1000
```

Comment:
Is description missing here?

Reply:
Because this page is the index (of sorts), I thought it might be best to leave the description off.